cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -1674,7 +1680,8 @@ private String generatePerBrokerConfiguration(NodeRef node, KafkaPool pool, Map<
                         )
                         .withAuthorization(cluster, authorization)
                         .withCruiseControl(cluster, ccMetricsReporter, node.broker())
-                        .withUserConfiguration(configuration, node.broker() && ccMetricsReporter != null);
+                        .withUserConfiguration(configuration, node.broker() && ccMetricsReporter != null)
+                        .withTieredStorage(cluster, tieredStorage);
```

Comment:
Maybe you can call this before the user configuration to have the user configuration at the end of the log.

Reply:
Done